CSOAR-590 Align CSOAR API documentation (#3627)
SumoLogic/sumologic-documentation: docs/cloud-soar/cloud-soar-apis.md

import useBaseUrl from '@docusaurus/useBaseUrl';
import ApiIntro from '../reuse/api-intro.md';

The Cloud SOAR APIs allow you to manage incidents, triage, and other Cloud SOAR features.

:::info

> This article applies to your organization *only if* your instance URL is a subdomain of `soar.sumologic.com`.
> Otherwise, please refer to:

Reviewer: ```suggestion
This article only applies to organizations with URL instance that have a subdomain of `soar.sumologic.com`. If it doesn't, refer to:
```

Author: Thanks for the review.
I rewrote the sentences to be similar and to better clarify the situation.
To add more context:
We have customers with "legacy" Cloud SOAR instances that have been provisioned with URLs having a subdomain of "soar.sumologic.com". These customers could have standard Sumo Logic instances with regional and custom alias or could not have any Sumo Logic instance at all. So the rule to identify which part of the document applies to "Delivery 1" instances is to check if the Cloud SOAR application has a URL in the browser URL bar matching the "*.soar.sumologic.com" pattern (despite the their actual Sumo Logic organization URL).

Hope that the new version of the text is better from your point of view.
Thanks

> * [Cloud SOAR APIs](/docs/api/cloud-soar) for the endpoints by geographic location
> * [API Authentication](/docs/api/getting-started#authentication) for details about API best practises.

Reviewer: ```suggestion
* [Cloud SOAR APIs](/docs/api/cloud-soar) for the endpoints by geographic location.
* [API Authentication](/docs/api/getting-started#authentication) for details about API best practices.
```

:::

### Inline API documentation

> For organizations having Cloud SOAR available at URL pattern `*.soar.sumologic.com`,
> the API documentation is available through your Cloud SOAR instance at the following URL:

Reviewer: ```suggestion
For organizations with a Cloud SOAR API URL pattern of `*.soar.sumologic.com`, you can find API documentation is through your Cloud SOAR instance at:
```

Author: Thanks for the review.
I rewrote the sentences to be similar and to better clarify the situation.
To add more context:
We have customers with "legacy" Cloud SOAR instances that have been provisioned with URLs having a subdomain of "soar.sumologic.com". These customers could have standard Sumo Logic instances with regional and custom alias or could not have any Sumo Logic instance at all. So the rule to identify which part of the document applies to "Delivery 1" instances is to check if the Cloud SOAR application has a URL in the browser URL bar matching the "*.soar.sumologic.com" pattern (despite the their actual Sumo Logic organization URL). 
So the URL to check is not only the API one but any URL the customers use to reach Cloud SOAR (from browser too).

Hope that the new version of the text is better from your point of view.
Thanks

### Generate an API token

> The REST API can only be used by authorized users who possess a valid JWT API key. An administrator should perform the
> following steps to issue an API key for an Cloud SOAR user:

Reviewer: ```suggestion
To use our Cloud SOAR REST API, you must be an authorized user with a valid JWT API key. Org administrators should perform the following steps to issue an API key for a Cloud SOAR user:
```

### Generate an API token

The REST API can only be used by authorized users who possess a valid JWT API key. An administrator should perform the
following steps to issue an API key for an Cloud SOAR user:

1. Select the desired user in the [User Management](/docs/cloud-soar/global-functions-menu/#user-management) section.
1. Scroll to the **API Token** section.
1. Click the **+** button on the right-hand side of the screen.

> 1. Click **Generate** to confirm the generation of the API token. The new JWT token will be found in the **API Token**
>    section of the user’s settings.

Reviewer: ```suggestion
1. Click **Generate** to confirm the generation of the API token. You can find your new JWT token in the **API Token** section of your user settings.
```

> A Cloud SOAR user can use the token displayed in the **API Token** section to provide authorization for their API calls.
> An administrator can revoke a user’s API token at any time by clicking the trash can icon to the far right side of the
> token row.

Reviewer: ```suggestion
You can use the token displayed in the **API Token** section to provide authorization for your API calls. Your org  administrator can revoke a user API token at any time by clicking the trash can icon to the far right side of the token row.
```
